src/appmixer/utils/timers/SchedulerTrigger/SchedulerTrigger.js

```
@@ -0,0 +1,233 @@
+                }
+                break;
+            case 'weeks':
+                const daysOfWeekMoments = daysOfWeek.map(day => baseDate.clone().set({
+                    hour, minute, second: 0, millisecond: 0
+                }).day(day.toLowerCase()));
+
+                daysOfWeekMoments.sort((a, b) => a.diff(baseDate) - b.diff(baseDate));
+                nextRun = daysOfWeekMoments.find(day => day.isAfter(baseDate)) || daysOfWeekMoments[0].add(1, 'week');
+                break;
+            case 'months':
+                const isLastDay = daysOfMonth.includes('last day of the month');
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)